playbooks/tasks/portal-role-task.yml

```diff
@@ -261,6 +255,17 @@
 # TODO: Log address
 
 # TODO: Should we setup allowance just once?
+
+# Set Sia allowance
+
+- name: Check if allowance is already set
+  ansible.builtin.command: docker exec sia siac renter allowance
+  register: get_allowance_result
+
+- name: Set allowance is not set flag (allowance = 0 SC)
+  set_fact:
+    allowance_is_not_set: '{{ get_allowance_result.stdout is match(".*\nAllowance:\s+0 SC.*") | bool }}'
```

Comment:
I think this might be `0 H` unless you tested this.

Reply:
1. It needs to be `0 H`.
2. It needs to match more text. This is the output:
```
$ docker exec sia siac renter allowance
Allowance:
  Amount:               20 KS
  Period:               8640 blocks
  Renew Window:         4320 blocks
  Hosts:                50
```

Reply:
Tested and updated, fresh sia contains `Allowance:      0 SC (No current allowance)`.

Reply:
> 1. It needs to be `0 H`.
> 
>     2. It needs to match more text. This is the output:
> 
> 
> ```
> $ docker exec sia siac renter allowance
> Allowance:
>   Amount:               20 KS
>   Period:               8640 blocks
>   Renew Window:         4320 blocks
>   Hosts:                50
> ```

I have changed the command to `docker exec sia siac renter`. The output is then (including extra newlines, the update will work with or without them):
```

Allowance:      0 SC (No current allowance)

File Summary:
  Files:                0
  Total Stored:         0  B
  Total Renewing Data:  0  B
Repair Status:
  Last Health Check:       153722867m
  Repair Data Remaining:   0  B
  Stuck Repair Remaining:  0  B
  Stuck Chunks:            0
  Max Health:              100%
  Min Redundancy:          -
  Lost Files:              0
Contract Summary:
  Active Contracts:    0
  Passive Contracts:   0
  Disabled Contracts:  0

```